src/explore-education-statistics-common/src/modules/find-statistics/components/ContentBlockRenderer.tsx

```diff
@@ -46,7 +46,15 @@ const ContentBlockRenderer = ({
 
   switch (type) {
     case 'MarkDownBlock':
-      return <ReactMarkdown className="dfe-content">{body}</ReactMarkdown>;
+      return (
```

Comment:
Just for info - we don't allow markdown blocks to be created any more, I think this is still in there in case there are still some old ones lurking. But good to have the check for italics here regardless.